src/components/MemberList.tsx

```diff
@@ -0,0 +1,33 @@
+import { useAuth0 } from "@auth0/auth0-react";
+import { IonItemSliding, IonItem, IonAvatar, IonImg, IonLabel, IonItemOptions, IonItemOption, IonIcon, IonNote } from "@ionic/react";
+import { closeOutline } from "ionicons/icons";
+
+interface MemberListProps {
+    users?: [] // should have all the users that are gonna appear
```

Comment:
add member pfp urls here as well? do we need to do that or will it just magically do it

Reply:
so in actuality, this has to be changed where it is an array of objects since an user object holds all the info that we need

Reply:
ok cool